### PR TITLE
add eslint/prettier to project

### PR DESCRIPTION
**Pull request by emilsvennesson**

### CODE REVIEW

File: app/tsconfig.json [hunk 1]
```diff
@@ -1,11 +1,7 @@
 {
   "compilerOptions": {
     "target": "es5",
```
Comment (EliasFalk) [inline app/tsconfig.json:3]: Should probably change `"target"` to `"es6"` or `"es2019"`.

File: app/tsconfig.json [hunk 2]
```diff
@@ -20,7 +16,5 @@
     "noEmit": true,
     "jsx": "react-jsx"
```
Comment (EliasFalk) [inline app/tsconfig.json:17]: Where is my boy `"strict": true`

**Comment by EliasFalk** [comment]
[eslint-config-airbnb-typescript](https://www.npmjs.com/package/eslint-config-airbnb-typescript)

Maybe worth a look? Not really sure what it improves upon but it says "_Enhances Airbnb's ESLint config with TypeScript support_" so must be good, right??

File: app/tsconfig.json (hunk 2 above)
Comment (EliasFalk) [reply app/tsconfig.json:17]: whoops forgot to expand code

**Comment by emilsvennesson** [comment]
> [eslint-config-airbnb-typescript](https://www.npmjs.com/package/eslint-config-airbnb-typescript)
> 
> Maybe worth a look? Not really sure what it improves upon but it says "_Enhances Airbnb's ESLint config with TypeScript support_" so must be good, right??

say no more fam

**Pull request merged**